src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/utils/format.ts

```diff
@@ -21,7 +21,7 @@ export function formatPhoneNumber(input: string): string {
   return input;
 }
 
-export function formatDate(inputDate: string, toFormat: DateFormat = "yyyy-MM-dd") {
+export function formatDate(inputDate: string, toFormat: string = "yyyy-MM-dd") {
```

Comment:
minor comment - can't we just add LLL dd, yyyy t 

As one of the valid dateFormats for typescript? in format.d.ts. 

type DateFormat = "yyyy-MMM-dd" | "yyyy-MM-ddZhh:mm:ss" | "yyyy-MM-dd";